### PR TITLE
Add command line interface to predict model

### PR DESCRIPTION
**Pull request by andrasschin**

### CODE REVIEW

File: dog_breed_identification/predict_model.py [hunk 1]
```diff
@@ -2,10 +2,13 @@
...
 import pandas as pd
+import click
 from dog_breed_identification.models.model import Model
 
-
-def predict(model: torch.nn.Module, directory: str, return_label: bool = True) -> None:
+@click.command()
+@click.argument("model_params")
+@click.argument("directory")
+def predict(model_params: torch.nn.Module, directory: str, return_label: bool = True) -> None:
```
Comment (madiistvan) [inline dog_breed_identification/predict_model.py]: Pls import Module from torch.nn for better readibility.

**Review by madiistvan — APPROVED** [review]
lgtm

**Review by madiistvan — APPROVED** [review]
lgtm

File: dog_breed_identification/predict_model.py (hunk 1 above)
Comment (andrasschin) [reply dog_breed_identification/predict_model.py]: fixed

**Pull request merged**